build.gradle.kts

```diff
@@ -1,23 +1,128 @@
+import com.android.build.api.dsl.extension.AndroidExtension
+import com.android.build.gradle.AppPlugin
+import com.android.build.gradle.BaseExtension
+import com.android.build.gradle.LibraryPlugin
+import com.android.builder.model.AndroidLibrary
+import com.jfrog.bintray.gradle.BintrayExtension
+import org.gradle.api.internal.plugins.DslObject
+import org.gradle.api.publish.maven.MavenPom
+import kotlin.properties.Delegates
+
 buildscript {
+    val kotlinVersion: String by project
+
     repositories {
         jcenter()
         google()
     }
 
     dependencies {
-        classpath(kotlin("gradle-plugin", version = properties["kotlinVersion"].toString()))
+        classpath(kotlin("gradle-plugin", kotlinVersion))
         classpath("com.android.tools.build:gradle:3.1.1")
+        classpath("com.github.dcendents:android-maven-gradle-plugin:2.1")
+        classpath("com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.0")
     }
 }
 
-subprojects {
-    repositories {
+group = "com.avito.ui-testing"
+version = "0.1.0"
+
+val minSdk: String by project
+val targetSdk: String by project
+
+fun Project.setupAndroidExtension() {
```

Comment:
Let's write it on every module with injecting `minSdk` and `targetSdk`. Now it looks a little bit confusing for me